### DISCUSSION TITLE
How can I manipulate the focus order that will be followed using the Tab key (or in reverse on shift-Tab)?

### DISCUSSION THREAD

**Discussion by reriley** [flet-dev/flet#2747] [Q&A]
### Question

How can I manipulate the focus order that will be followed using the Tab key (or in reverse on shift-Tab)? The automatic behavior seems to left to right and top to bottom across the entire page, rather than remaining within the current container, for example. Thus when tabbing through a form it keeps jumping over to the side menu that I have. Thanks for the tips.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by emojiiii** [comment]
https://flet.dev/docs/guides/python/accessibility

**Reply by reriley** [reply]
Thank you. I think you are getting at the ability to intercept a keyboard event and administer a focus order that responds to tab or shift-tab keystrokes. 

For the record, the simpler solution that I settled on was to redesign the gui and make sure I disable any controls not currently in the area of focus.
